src/components/CPageCard/index.tsx

```diff
@@ -1,20 +1,38 @@
 import React from 'react';
+import cn from 'classnames';
 
 import CCard from '../CCard';
 
-const CPageCard = () => {
+interface CPageCard {
+  divider?: boolean;
+  title?: JSX.Element | React.ReactNode;
+  children?: JSX.Element | React.ReactNode;
+  className?: string;
+}
+
+const CPageCard = ({ divider, title, children, className, ...props }: CPageCard) => {
+  let dividerStyle = '';
+  let padding = '';
+
+  if (divider) {
+    dividerStyle = 'border-b border-[rgba(5, 1, 66, 0.10)] mb-4';
+    padding = 'pl-2';
+  } else {
+    dividerStyle = 'border-none mb-0';
+    padding = 'p-0';
+  }
   return (
     <CCard
-      className="flex justify-between w-[1000px] max-w-full px-8 py-5"
+      className={cn('flex flex-col justify-between w-full h-full', className)}
       bgColor="#fff"
       borderColor="rgba(5, 1, 66, 0.10)"
+      {...props}
     >
-      <div className="w-full pb-2 border-b border-[rgba(5, 1, 66, 0.10)]">{title}</div>
+      <div className="w-full">{title}</div>
```

Comment:
Check if `title` is not undefined or null, then use it inside the component.

Something like `{title && <......}`